### ISSUE TITLE
Add Ska.engarchive 4.45.1

### ISSUE THREAD

**Issue by taldcroft** [sot/skare3#207]
Fixes a problem in testing on 32-bit
```
#175: Skip sync tests on 32-bit platform (version 4.45.1)
```
```
#172: Fix PEP8 and flake8 issues and add flake8 checking
```

**Comment by jeanconn** [comment]
Oops, I should have left this one open to track the MATLAB label

**Comment by jeanconn** [comment]
Closed by #206
